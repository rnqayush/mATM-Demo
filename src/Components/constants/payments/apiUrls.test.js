import PAYMENTS_URLS from './apiUrls';

jest.mock('@optimus/core-feature-toggle');

describe('payments urls', () => {
  it('should return all urls ', () => {
    const actualUrls = PAYMENTS_URLS();
    expect(actualUrls).toMatchSnapshot();
  });
});
