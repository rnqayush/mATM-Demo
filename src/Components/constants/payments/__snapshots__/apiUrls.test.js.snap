// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`payments urls should return all urls  1`] = `
Object {
  "ACCOUNTS": "https://api.dev-opt.idfcfirstbank.com/api/payments/v1/accounts",
  "ADD_FUNDS": "https://api.dev-opt.idfcfirstbank.com/api/upi/v1/collect-money/add-funds",
  "APPROVALS_FILTERS": "https://api.dev-opt.idfcfirstbank.com/api/bas/v1/user/transactions/history/filters",
  "APPROVALS_TRANSACTION_ACTION": "https://api.dev-opt.idfcfirstbank.com/api/bas/v2/authorised-transaction/{transactionType}/{transactionId}",
  "APPROVALS_TRANSACTION_ACTION_V2": "https://api.dev-opt.idfcfirstbank.com/api/approvals/v1/transaction/{transactionType}/{transactionId}",
  "APPROVALS_TRANSACTION_HISTORY": "https://api.dev-opt.idfcfirstbank.com/api/bas/v1/user/transactions/history",
  "AUTO_SELECT": "https://api.dev-opt.idfcfirstbank.com/api/payments/v1/fundtransfer/railroads/autoselect",
  "BANKACCOUNTS": "https://api.dev-opt.idfcfirstbank.com/api/upi/v1/bankAccounts",
  "BANKS": "https://api.dev-opt.idfcfirstbank.com/api/payments/v1/banks",
  "BANK_BRANCHES": "https://api.dev-opt.idfcfirstbank.com/api/payments/v1/banks/{bankId}/branches",
  "BLOCKED_VPA_LIST": "https://api.dev-opt.idfcfirstbank.com/api/upi/v1/list-blocked-vpa",
  "CHANGE_PIN": "https://api.dev-opt.idfcfirstbank.com/api/upi/v1/changePIN",
  "CHECK_BALANCE": "https://api.dev-opt.idfcfirstbank.com/api/upi/v1/check-balance",
  "COLLECT_MONEY": "https://api.dev-opt.idfcfirstbank.com/api/upi/v1/collect-money",
  "COLLECT_MONEY_REQUESTS": "https://api.dev-opt.idfcfirstbank.com/api/upi/v1/collect-money-request",
  "CONFIG": "https://api.dev-opt.idfcfirstbank.com/api/payments/v1/configurations",
  "CONFIRM_COLLECT": "https://api.dev-opt.idfcfirstbank.com/api/upi/v1/confirm-collect",
  "DOWNLOAD_BILLPAY_RECEIPT": "https://api.dev-opt.idfcfirstbank.com/api/billpay/v1/bill/receipt",
  "DOWNLOAD_FUND_TRANSFER_RECEIPT": "https://api.dev-opt.idfcfirstbank.com/api/payments/v1/fundtransfer/receipt",
  "EXTERNAL_FUND_TRANSFER": "https://api.dev-opt.idfcfirstbank.com/api/upi/v1/external-fundtransfer",
  "GENERATE_OTP": "https://api.dev-opt.idfcfirstbank.com/api/upi/v1/generate-otp",
  "GET_DISPUTE_OPTIONS_FOR_BILLPAY": "https://api.dev-opt.idfcfirstbank.com/api/billpay/v1/dispute/options",
  "GET_DISPUTE_OPTIONS_FOR_UPI": "https://api.dev-opt.idfcfirstbank.com/api/upi/v1/dispute/options",
  "GET_PUBLIC_KEYS": "https://api.dev-opt.idfcfirstbank.com/api/upi/v1/npci/list-public-keys",
  "GET_UPI_RAILROAD_DETAILS": "https://api.dev-opt.idfcfirstbank.com/api/upi/v1/railroad-details",
  "INITIATE_BINDING": "https://api.dev-opt.idfcfirstbank.com/api/upi/v1/initiate-binding",
  "INITIATE_INWARD_REMITTANCE": "https://api.dev-opt.idfcfirstbank.com/api/payments/v1/inward-remittance/payload/encrypt",
  "IS_FUNDTRANSFER_ALLOWED": "https://api.dev-opt.idfcfirstbank.com/api/beneficiary/v1/payee/fundtransfer-allowed",
  "LINKED_BANK_ACCOUNTS": "https://api.dev-opt.idfcfirstbank.com/api/upi/v1/linkedAccounts",
  "LINK_ACCOUNT_TO_VPA": "https://api.dev-opt.idfcfirstbank.com/api/upi/v1/link-account",
  "NRO_TO_NRE_TRANSFER": "https://api.dev-opt.idfcfirstbank.com/api/payments/v1/nro-to-nre-transfer/payload/encrypt",
  "PAYEE_ADDITION_ALLOWED": "https://api.dev-opt.idfcfirstbank.com/api/beneficiary/v1/isPayeeAdditionAllowed",
  "PAYEE_LIMIT": "https://api.dev-opt.idfcfirstbank.com/api/payments/v1/fundtransfer/limits/payee-limit",
  "PAYEE_SEARCH": "https://api.dev-opt.idfcfirstbank.com/api/beneficiary/v1/payee/search",
  "PAY_ABROAD": "https://api.dev-opt.idfcfirstbank.com/api/payments/v1/pay-abroad/payload/encrypt",
  "PAY_BILL_WITH_REVERSAL": "https://api.dev-opt.idfcfirstbank.com/api/billpay/v2/bill/pay",
  "RAILROAD": "https://api.dev-opt.idfcfirstbank.com/api/payments/v1/fundtransfer/railroads",
  "RAISE_DISPUTE_FOR_UPI": "https://api.dev-opt.idfcfirstbank.com/api/upi/v1/dispute",
  "RAISE_DISPUTE_FOR_UPI_TRANSACTION": "https://api.dev-opt.idfcfirstbank.com/api/upi/v1/dispute/raise",
  "RECEIVED_APPROVALS_FILTERS": "https://api.dev-opt.idfcfirstbank.com/api/bas/v1/user/transactions/approval/filters",
  "RECURRING_TRANSACTIONS": "https://api.dev-opt.idfcfirstbank.com/api/payments/v2/recurring-transactions",
  "REMOVE_ACCOUNT_FROM_VPA": "https://api.dev-opt.idfcfirstbank.com/api/upi/v1/delink-account",
  "SAVE_NEW_PAYEE_DETAILS": "https://api.dev-opt.idfcfirstbank.com/api/payments/v1/transaction/new-payee",
  "SAVE_QR_DETAILS": "https://api.dev-opt.idfcfirstbank.com/api/upi/v1/verifyvpa/map-transaction-details",
  "SEARCH": "https://api.dev-opt.idfcfirstbank.com/api/payments/v1/search",
  "SEND_SILENT_MESSAGE": "https://api.dev-opt.idfcfirstbank.com/api/upi/v1/send-silent-message",
  "SETUP_VPA": "https://api.dev-opt.idfcfirstbank.com/api/upi/v1/setup-vpa",
  "SET_ACCOUNT_AS_PRIMARY": "https://api.dev-opt.idfcfirstbank.com/api/upi/v1/setAccountAsPrimary",
  "SET_OR_FORGOT_PIN": "https://api.dev-opt.idfcfirstbank.com/api/upi/v1/set-or-forgot-pin",
  "TRANSACTION_AUDIT": "https://api.dev-opt.idfcfirstbank.com/api/bas/v1/user/transactions/audit",
  "TRANSACTION_AUDIT_V2": "https://api.dev-opt.idfcfirstbank.com/api/approvals/v1/transactions/audit",
  "TRANSACTION_DETAILS": "https://api.dev-opt.idfcfirstbank.com/api/payments/v1/transaction",
  "UPI_BANKS": "https://api.dev-opt.idfcfirstbank.com/api/upi/v1/banks",
  "UPI_INCOMING_REQUESTS": "https://api.dev-opt.idfcfirstbank.com/api/upi/v1/incoming-requests",
  "UPI_INCOMING_REQUESTS_VISITED": "https://api.dev-opt.idfcfirstbank.com/api/upi/v1/incoming-requests/visited",
  "UPI_TRANSACTIONS": "https://api.dev-opt.idfcfirstbank.com/api/upi/v1/transactions",
  "VALIDATE_DEVICE_BINDING": "https://api.dev-opt.idfcfirstbank.com/api/upi/v1/validate-device-binding",
  "VALIDATE_LIMIT": "https://api.dev-opt.idfcfirstbank.com/api/payments/v1/fundtransfer/limits/validate",
  "VALIDATE_LIMIT_UPI_24_HOUR": "https://api.dev-opt.idfcfirstbank.com/api/upi/v1/validate/limits/24hours",
  "VALIDATE_LIMIT_UPI_24_HOUR_PAYMENTS": "https://api.dev-opt.idfcfirstbank.com/api/payments/v1/upi/validate/limits/24hours",
  "VALIDATE_LIMIT_V2": "https://api.dev-opt.idfcfirstbank.com/api/payments/v2/fundtransfer/limits/validate",
  "VALIDATE_NUMBER_OF_TRANSACTIONS_LIMIT": "https://api.dev-opt.idfcfirstbank.com/api/upi/v1/validate-collect-request",
  "VALIDATE_PER_DAY_LIMIT": "https://api.dev-opt.idfcfirstbank.com/api/upi/v1/collect/validate/limits/per-day",
  "VALIDATE_PER_TXN_LIMIT": "https://api.dev-opt.idfcfirstbank.com/api/upi/v1/collect/validate/limits/per-transaction",
  "VERIFY_VPA": "https://api.dev-opt.idfcfirstbank.com/api/upi/v1/verifyvpa",
  "VPA": "https://api.dev-opt.idfcfirstbank.com/api/upi/v1/vpa",
  "VPA_HANDLES_LIST": "https://api.dev-opt.idfcfirstbank.com/api/upi/v1/handles",
}
`;
